src/tool/gene-keybind-markdown.js

```diff
@@ -43,14 +62,12 @@ rd.on('line', function(line) {
         intellij = getIntelliJ(line);
         
         var format = available(escape(key)) + ' | ' + available(escape(mac)) + ' | ' + intellij + ' | ';
-        if (hasCommand(command)) {
-            console.log(format + '✅');
-        } else {
-            console.log(format + 'N/A');
-        }
+        commandsForHeader.set(format, hasCommand(command) || commandsForHeader.get(format));
```

Comment:
We could also be strict and throw an error if there is a version of a command that is '✅' and one that is 'N/A'. Let me know what you would prefer.